src/main/java/pl/confitura/jelatyna/email/EmailService.java

```diff
@@ -0,0 +1,21 @@
+package pl.confitura.jelatyna.email;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.confitura.jelatyna.admin.Admin;
+
+@Service
+public class EmailService {
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Autowired
+    private Sender sender;
+
+    public void adminCreated(Admin admin) {
+        sender.send(admin.getEmail(), "admin-creation",
```

Comment:
good practise...
ale ok... jeśli nie pole statyczne, to może pole które będzie wstrzykiwane z propertisów?